packages/ia-components/sandbox/track-lists/track.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Creates the track title to display based on track object
+ *
+ * @param { string } name
+ * @param { string } title
+ * @param { string } artist
+ * @param { string } creator
+ * @param { boolean } isAlbum
+ *
+ * @return { string or react fragment } trackTitle
+ */
+const parseTrackTitle = ({
+  name = '',
+  title = '',
+  albumCreator,
+  albumName,
+  creator = '',
+  artist = '',
+  isAlbum = false
+}) => {
+  if (isAlbum) { return 'Full album'; }
+  const albumArtist = Array.isArray(albumCreator) ? albumCreator.join('; ') : albumCreator;
+  const trackArtist = creator || artist;
+
+  /* this considers "Best of" albums */
+  const titleHasTrackArtist = albumName.includes(trackArtist);
+  const isMainAlbumArtist = titleHasTrackArtist || (trackArtist === albumArtist);
+  const artistName = isMainAlbumArtist ? '' : trackArtist;
+  const titleArtistDelimiter = artistName ? ' - ' : '';
+
+  if (title) {
+    return (
+      <Fragment>
```

Comment:
With React 16, you can now just use `<>` and `</>` for fragments.

Reply:
Yep.

Reply:
Going to hold for now, my editor is borking when using shorthand.  On the next update, i will also update this.